src/less/tooltip/ds6/tooltip.less

```diff
@@ -0,0 +1,201 @@
+@import "../../less/ds6/variables.less";
+@import "../../less/ds6/mixins.less";
+
+@import "tooltip-mixins.less";
+@import "../base/tooltip.less";
+
+.tooltip-comp {
+    .tooltip,
+    .tourtip {
+        background-color: @ds6-color-p002-blue;
+        color: @ds6-color-g201-gray;
+    }
+
+    .infotip {
+        background-color: @ds6-color-g201-gray;
+        box-shadow: 0 0 5px 1px @ds6-color-g205-gray;
```

Comment:
This box shadow seems a bit heavy. Let's ask design to get us the actual box shadow they used in a Zeplin project or something.